src/main/scala/uk/gov/hmrc/integration/ServiceSpec.scala

```diff
@@ -35,33 +35,40 @@ trait ServiceSpec
   this: TestSuite =>
 
   override def fakeApplication(): Application =
-    GuiceApplicationBuilder(environment = Environment.simple(mode = applicationMode)).configure(configMap).build()
+    GuiceApplicationBuilder(environment = Environment.simple(mode = applicationMode.getOrElse(Mode.Dev)))
```

Comment:
If we're explicitly setting `applicationMode` to None for configuration, should the default revert to `Test` (which is the default for GuiceApplicationBuilder)?

Reply:
Makes sense